frontend/src/components/Entities.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div id="root-element">
+    <entitiesList></entitiesList>
+
+  </div>
+</template>
+
+<script>
+import EntitiesList from './EntitiesList.vue'
+
+export default {
+  name: 'entities',
+  components: {
+    'entitiesList': EntitiesList
+  },
+  data () {
+    return {
+      entities: [],
+    }
+  },
+  methods: {
+    loadEntities: function() {
+      var self = this
```

Comment:
Vue que tu utilises ES6, tu n'a plus besoin de sauvegarder le contexte avec ``var  self = this``.